src/chains/supported.chains.ts

```diff
@@ -181,11 +181,11 @@ export const supportedEVMChains: EVMChain[] = [
       'https://raw.githubusercontent.com/sushiswap/default-token-list/master/tokens/okex.json',
     multicallAddress: multicallAddresses[ChainId.OKT],
 
-    // https://okexchain-docs.readthedocs.io/en/latest/developers/quick-start-for-mainnet.html
+    // https://okc-docs.readthedocs.io/en/latest/developers/quick-start-for-mainnet.html
     metamask: {
       chainId: prefixChainId(66),
-      blockExplorerUrls: ['https://www.oklink.com/okexchain/'],
-      chainName: 'OKExChain Mainnet',
+      blockExplorerUrls: ['https://www.oklink.com/en/okc/'],
+      chainName: 'OKC Mainnet',
```

Comment:
The metamask definition should match https://chainlist.org/chain/66, as Metamask compares the information when adding new chains and warns the user if we pass another name (please try if that is still the case).

Reply:
Ah good to know thanks! I updated it and tested it. Before it showed the warning, now it doesnt :)